model-optimizer/extensions/front/instance_normalization.py

```diff
@@ -29,18 +35,20 @@ class InstanceNormalization(FrontReplacementOp):
     enabled = True
 
     def replace_op(self, graph: Graph, node: Node):
-
+        name = node.soft_get('name', node.id)
         # Add new nodes
-        mvn = MVN(graph, {'eps': node.epsilon, 'name': node.name + '/Ins_Norm/MVN_', }).create_node()
-        mul = Mul(graph, {'axis': 1, 'name': node.name + '/Ins_Norm/mul_'}).create_node()
-        add = Add(graph, {'axis': 1, 'name': node.name + '/Ins_Norm/add_'}).create_node()
-
-        # Connect nodes
-        node.in_port(0).get_connection().set_destination(mvn.in_port(0))
-        node.in_port(1).get_connection().set_destination(mul.in_port(1))
-        node.in_port(2).get_connection().set_destination(add.in_port(1))
-
-        mvn.out_port(0).connect(mul.in_port(0))
-        mul.out_port(0).connect(add.in_port(0))
+        rank = Rank(graph, {'name': name + '/Rank'}).create_node()
+        rng = create_op_with_const_inputs(graph, Range, {0: int64_array(2), 2: int64_array(1)},
+                                          {'name': name + '/Range', 'output_type': np.int64})
+        mvn = MVN(graph, {'eps': node.epsilon, 'eps_mode': 'inside_sqrt', 'normalize_variance': 1,
+                          'name': name + '/Ins_Norm/MVN_', }).create_node([node.in_port(0).get_source().node, rng])
```

Comment:
I think you also need to make a comment about this block. And please say in the comment that normalization is not across channel dimension.
BTW, may we have a tensor of 2 or less rank?

Reply:
Rank can't be less than 4 based on ONNX spec https://github.com/onnx/onnx/blob/master/docs/Operators.md#InstanceNormalization